src/fetch.js

```diff
@@ -38,7 +38,10 @@ module.exports = async ({
  */
 const clean = item => {
   forEach(item, (value, key) => {
-    if (startsWith(key, `__`)) {
+    if (endsWith(key, `component`)) {
```

Comment:
Is it necessary `__`?

Reply:
We only care about `__component`, so I'd say `endsWith` is both redundant and could potentially mess up other fields if they should end in "component". Not sure this would ever happen, but it's an unnecessary risk.
```suggestion
    if (key === `__component`) {
```